coregistration/Coregistration.ipynb

```diff
@@ -82,14 +82,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "class MaxCCPredicate:\n",
-    "    def __init__(self, maxcc):\n",
-    "        self.maxcc = maxcc\n",
+    "class MaxCloudCoveragePredicate:\n",
+    "    def __init__(self, max_cloud_coverage):\n",
+    "        self.max_cloud_coverage = max_cloud_coverage\n",
     "\n",
-    "    def __call__(self, img_cm):\n",
-    "        w, h, _ = img_cm.shape\n",
-    "        cc = np.sum(img_cm) / (w * h)\n",
-    "        return cc <= self.maxcc"
+    "    def __call__(self, img_cloud_mask):\n",
```

Comment:
`img_cloud_mask` could be changed to `cloud_mask`, especially if you add annotations so that it's clear that it should be a numpy array